src/xml/namespace.cr

```diff
@@ -4,8 +4,8 @@ struct XML::Namespace
   def initialize(@document : Node, @ns : LibXML::NS*)
   end
 
-  def hash
-    object_id
+  def hashme(hasher)
```

Comment:
Could this perhaps be generalized? Like if a type has method `object_id`, use this for calculating the hash, whether it is a reference or not.